pkgs/dart_mcp_server/CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-# Version 0.1.0 (Dart SDK 3.8.0)
+# 0.1.1-wip
+
+* Instruct clients to prefer MCP tools over running tools in the shell.
```

Comment:
This can actually move into the below version, I plan to do a roll soon so it will make it into the 3.8 SDK.

The versions here are weird since its not actually published but I did want some sort of changelog.